wifistats/index.js

```diff
@@ -0,0 +1,117 @@
+const exec    = require('child_process').exec;
+const async   = require('async');
+// The tools
+const airport = require('./airport');
+const iwconfig  = require('./iwconfig');
```

Comment:
note that `iwconfig` is deprecated (has been for quite a while) and `ip` is its successor.

Reply:
Shit, it has totally different output :( I will look at that. Thanks.

Reply:
Maybe using directly /proc/net/wireless will be better

Reply:
Using /proc/net/wireless now.